System - Add Wireless Set No38 radio and 12ft antenna

Spartan190 opened the pull request:
**When merged this pull request will:**
- add the Wireless Set No38 Radio
- add the Wireless Set No38 Radio 12ft antenna
- add the WS19 vehicle rack

Video: https://youtu.be/RMP1mhgx4UI
![image-85](https://github.com/IDI-Systems/acre2/assets/5320936/ef9194bb-5a22-49d3-9604-bdacdb0b964f)
![AntennaAndBag](https://github.com/IDI-Systems/acre2/assets/5320936/1fbc3b7e-9071-4921-98ee-0e378adbc146)
![AntennaConnect](https://github.com/IDI-Systems/acre2/assets/5320936/7803a3e8-484c-40a5-90ce-d6601e31fe3e)

Spartan190 committed:
Add:
   - initial integration of WS38 based on PRC343

Spartan190 committed:
Rename:
   - renamed prc343_RadioDialogue to ws38_RadioDialogue

Add:
   - added dumy mode knob
   - added functionality to turn radio on and off
   - added updating of the knob when sending

Spartan190 committed:
Fix:
   - disbabled debugging

Spartan190 committed:
Fix:
   - fixed modeknob button size and position

Add:
   - added starting of sending when on send mode

Spartan190 committed:
Add:
   - adding setting of frequencies with the dial (WIP)

Spartan190 committed:
Add:
   - added changing of frequencies

Spartan190 committed:
Fix:
   - fixed getting of function state

Remove:
   - removed updating of UI when transmitting (for now)

Spartan190 committed:
Fix:
   - fixed handling of UI update for sending

Spartan190 committed:
Update:
   - updated visual images

Fix:
   - fixed position and size of dial button
   - fixed radio to be on by default

Spartan190 committed:
Update:
   - updated mode knob for better visuals

Spartan190 committed:
Add:
   - added custom antennas for the WS38

Spartan190 committed:
Add:
   - added 12FT placable antenna

Spartan190 committed:
Fix:
   - fixed setting of frequencies
   - fixed the antenna gains

Spartan190 committed:
Fix:
   - fixed radio to be a pack radio

Spartan190 committed:
Add:
   - added correct model for WS38
   - added correct antenna model

Spartan190 committed:
Add:
   - added new sounds for starting and ending transmitting

Remove:
   - removed "beep"  when starting transmitting

Fix:
   - fixed display name

Spartan190 committed:
Fix:
   - fixed masses of WS38 and 12ft antenna
   - fix start transmission sound
   - WS38 is off by default

Spartan190 committed:
Add:
   - correct editor preview image
   - made the antenna carryable with ace

Fix:
   - fixed interaction location

Spartan190 committed:
Add:
   - added correct icon

Spartan190 committed:
Add:
   - added WS19 rack
   - added possibility to mount the WS38 to the WS19 rack

Spartan190 committed:
Add:
   - added Antenna bag icon
   - added Antenna bag model

Spartan190 committed:
Refactor:
   - removed old files of WS38
   - replaced tabs with spaces

jonpas commented:
Nice work! I will try to review this soon!

Mike-MF commented on addons/sys_rack/ws19/__PREP__.sqf:
```diff
@@ -0,0 +1,7 @@
+#include "..\script_component.hpp"
+
+TRACE_1("enter", _this);
```
```suggestion
TRACE_1("enter",_this);
```

Any Macros added shouldn't have spaces within them.

Mike-MF commented:
Needs to be updated from master so I can test it.

jonpas commented:
@Spartan190 Please allow pushing to this branch, or update to master yourself.

Spartan190 committed:
Update addons/sys_rack/ws19/__PREP__.sqf

Thanks, the auto format of VSCode is weird for macros

Co-authored-by: Mike-MF <TyroneMF@hotmail.com>

Spartan190 commented:
> @Spartan190 Please allow pushing to this branch, or update to master yourself.

I have invited you and @Mike-MF as members.
If there is a better way to grant access, let me know. It's actually my first time working with a fork

Spartan190 commented:
We have also been using this in some events and noticed that the range is not very good and also not really usable in areas with a lot of trees. But we are currently on it trying to get it fixed.

jonpas commented:
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

This is most ideal.

Spartan190 commented:
> https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork
> 
> This is most ideal.

Ok I just noticed I don't have access to change this in the PR.

Spartan190 commented:
I updated the branch with latest main

Mike-MF committed:
Remove macro padding

Mike-MF committed:
Fix header formatting, PTTUp/PTTDown

Mike-MF reviewed (approved):
Nothing wrong that I could see.

mrschick commented:
> > https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork
> > This is most ideal.
> 
> Ok I just noticed I don't have access to change this in the PR.

[Same old issue with organization PRs](https://github.com/orgs/community/discussions/5634) 😕

jonpas commented:
> We have also been using this in some events and noticed that the range is not very good and also not really usable in areas with a lot of trees. But we are currently on it trying to get it fixed.

What was the last status on signal quality on this radio? Has anything been done to mitigate it?

Spartan190 commented:
> > We have also been using this in some events and noticed that the range is not very good and also not really usable in areas with a lot of trees. But we are currently on it trying to get it fixed.
> 
> What was the last status on signal quality on this radio? Has anything been done to mitigate it?

We have been checking on it and have a version to test but we haven't been able to test it in the group yet.

Spartan190 committed:
Merge remote-tracking branch 'remotes/origin/master' into WirelessSetNo38

jonpas commented:
@Soldia1138 is looking into the antenna stuff and will lend a hand when time permits @Spartan190.

Otherwise this looks good to me for a merge. I heard some documentation is available as well? It would be awesome to get it included in this PR.

jonpas committed:
Merge branch 'master' into WirelessSetNo38

jonpas committed:
Fix missing newlines at EOF

Spartan190 committed:
Remove:
   - removed references to 343 images

Spartan190 committed:
Refactor:
   - changed or removed comments referencing the 343 radio

rautamiekka reviewed (changes requested):
Also let's not **continue to** mix units.

rautamiekka reviewed (changes requested):
EDIT: GitHub forced me to click 'Send review' twice, cuz the 1st click gave a JSON syntax error close to the HTML `<!DOCTYPE` tag, then the 2nd went through and double-posted, so GitHub has an overlooked bug.

Spartan190 committed:
Refactor:
   - refactored frequencies to use the frequencies of the AN/PRC-77 Radio
   - copied 77 antennas data to ArialRods
